code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -190,7 +191,8 @@
 /mob/living/carbon/human/attack_paw(mob/living/carbon/monkey/M)
 	var/dam_zone = pick("chest", "l_hand", "r_hand", "l_leg", "r_leg")
 	var/obj/item/bodypart/affecting = get_bodypart(ran_zone(dam_zone))
-
+	if(!affecting)
+		affecting = get_bodypart("chest")

```

Comment:
so we don't attack a missing limb. If we can't find the limb, we default to chest.